bfvmm/include/intrinsics/intrinsics_intel_x64.h

```diff
@@ -79,8 +84,18 @@ class intrinsics_intel_x64 : public intrinsics_x64
     virtual bool vmread(uint64_t field, uint64_t *val)
     { return __vmread(field, val); }
 
+    virtual uint64_t vmread(uint64_t field)
```

Comment:
I think that's a 'buyer beware' version of the function in my opinion. If you use it, you are taking on the risk that the read could have failed. A similar function is provided in the VMCS class which sets a global bool flag (which is essentially ignored until all errors have coalesced together). Without a function like this, the code size balloons, with it there are runtime risks. Perhaps there should be a policy or main direction for this trade off in general.